### DISCUSSION TITLE
ServiceOperationManifest documentation for custom connector extensions

### DISCUSSION THREAD

**Discussion by bgelens** [Azure/logicapps#830] [Feature Ideas/Requests]
I'm building a custom connector extension for internal use and I'm having a hard time il-spying the ServiceOperationManifest possibilities. For example, I require an input which generates a drop down (like the Enum but without the option to provide a custom input) and dependent on the selection, an additional required parameter is surfaced or not (similar to ConstraintDependentParameter in ConnectionString parameters).

My example is just one of the things I need. It would be nice to find some documentation but so far I've stranded.

Is there any public documentation available on what we can do here?

**Comment by earloc** [comment]
Yeah, getting this to work would be impossible without decompiling existing connectors, which is a real PITA.
